pkg/collect/goldpinger.go

```diff
@@ -26,6 +26,8 @@ import (
 	"k8s.io/utils/ptr"
 )
 
+var GoldpingerImage = "bloomberg/goldpinger:3.10.1"
```

Comment:
```suggestion
var goldpingerImage = "bloomberg/goldpinger:3.10.1"
```

No need to make this public if not necessary